submissions/AnnaStoyano/popup/style.css

```diff
@@ -0,0 +1,199 @@
+    justify-content: flex-end;
+    align-items: center;
+    list-style: none;
+    
+}
+
+header li:hover{
+    cursor: pointer;
+}
+
+li>a{
+    text-decoration: none;
+    color: rgb(75, 74, 74);
+}
+
+
+label.icon{
+    display: block;
+}
+
+li + li{
+    margin-left: 20px ;
+}
+
+.menu{
+    cursor: pointer;
+    width:25px;
+    height: 25px;
+    outline: none;
+    border: none;
+    padding: 0;
+    background-position: -2px -1803px ;
+    position: relative;
+}
+
+#main_menu{
+    display: none;
+}
+
+
+#main_menu:checked + .rectangle{
+   display: block;
+}
+
+.rectangle::before{
+
+    content: "";
+    width: 10px;
+    height: 10px;
+    border: 0.5px solid rgb(184, 182, 182);
+    position: absolute;
+    background-color: white;
+    top:-6px;
+    right: 80px;
+    transform: rotate(45deg); 
+    border-bottom: none;
+    border-right: none;
+}
+
+.rectangle {
+    display: none;
+    position: absolute;
+    top:65px;
+    right: 50px;
+    width: 300px;
+    background-color: white;
+    border:0.5px solid rgb(184, 182, 182);
+    box-shadow: 1px 1px 10px rgb(184, 182, 182); 
+}
+
+
+
+
+.submenu {
+    padding: 40px 25px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    flex-wrap: wrap;
+    
+}
+
+.acount{
+    width:47px;
+    height: 47px;
+    background-position: -10px -1603px ;
+}
+
+.circle {
+    width:26px;
+    height: 26px;
+    background-color: rgb(131, 130, 130);
+    border-radius: 13px;
+}
+
+.bell {
+    color: white;
+    background-position: 3px -2320px ;
+    filter:invert(100%);
+}
+
+.referens{
+    display: flex;
+    flex-direction: column; 
+    align-items: center;
+    width: 80px;
+    height: 80px;
+    text-align: center;
+    margin-top: 10px ;
+    cursor: pointer;
+    outline: none;
+   
+}
+
+.referens:focus {
+    
+    border:0.5px solid rgb(184, 182, 182);
+    box-shadow: 1px 1px 5px rgb(184, 182, 182);
+}
+
+.submenu .referens:first-child div.icon{
+   background-position: -4px -730px;
+}
+
+.submenu .referens:nth-child(2) div.icon{
+    background-position: -4px -2186px ;
+ }
+ 
+.submenu .referens:nth-child(3) div.icon{
+    background-position: -4px -420px;
+ }
+ 
+.submenu .referens:nth-child(4) div.icon{
+    background-position: -4px -1910px ;
+ }
+ 
+.submenu .referens:nth-child(5) div.icon{
+    background-position: -30px -2328px
+ }
+
+ .submenu .referens:nth-child(6) div.icon{
+    background-position: -4px -352px;
+ }
+ .submenu .referens:nth-child(7) div.icon{
+    background-position: -4px -2742px
+ }
+ .submenu .referens:nth-child(8) div.icon{
+    background-position: -4px -524px;
+ }
+ .submenu .referens:nth-child(9) div.icon{
+    background-position: -4px -178px;
+ }
+
+ .submenu .referens:nth-child(10) div.icon{
+    background-position: -4px -593px;
+ }
+
+ #more_m{
+    display: none;
+}
+
+#more_m + .submenu{
+    display: none;
+}
+
+#more_m:checked + .submenu{
+    display: flex;
+ }
+
+.more{

```

Comment:
Add some styles on `:hover` and `focus ` to all interactive elements.

Reply:
Thank you) I have tried to correct all you remarks